plugins/mcreator-localization/lang/texts.properties

```diff
@@ -291,6 +291,7 @@ dialog.imageeditor.stamp_tool_smooth_resizing=Smooth resizing
 dialog.imageeditor.stamp_tool_connect_points=Connect points
 dialog.imageeditor.stamp_tool_base_texture=Base texture:
 dialog.image_maker.save=Save this texture
+dialog.image_maker.save.view_only=<html>This is a view-only external image asset intended for exploratory use only.<br>Distribution of these files (in case copyrighted) may be prohibited by the license<br>included in the same archive as this asset. Therefore, it is highly recommended<br>for you to check the included license (if any) before continuing.
```

Comment:
(if any) may give user idea that no license means free use for any case, so I would reword this to make user more cautious.

I would also split this into two paragraphs as right now the message looks a bit crammed

Reply:
You mean splitting the UI string into more lines? Because the contained text is already divided into 4 of them.

Reply:
Yep, like two sections

> This is a view-only external image asset intended for exploratory use only.
> Distribution of these files (in case copyrighted) may be prohibited by the license
> included in the same archive as this asset. 
> 
> Therefore, it is highly recommended for you to check the included license (if any) 
> before continuing

(idea, but different text)

Reply:
Looks better?
![image](https://user-images.githubusercontent.com/71347607/235211833-c6a36ef0-7cd8-4aaf-8024-f2d3c087cb01.png)